src/openrct2/ride/Vehicle.cpp

```diff
@@ -9890,3 +9899,150 @@ void vehicle_invalidate_window(rct_vehicle * vehicle)
 
     window_invalidate(w);
 }
+
+void vehicle_update_crossings(rct_vehicle * vehicle)
```

Comment:
Looks like the argument could be const? It's passed down to `vehicle_get_{head|tail}()` which should be const already and `vehicle_claxon()` which I don't know